### ISSUE TITLE
golangci doesnt work with golang1.13

### ISSUE THREAD

**Issue by yfei1** [googleforgames/open-match#821]

**Comment by yfei1** [comment]
```bash
> GO111MODULE=on /usr/local/google/home/yufanfei/go/src/open-match.dev/open-match/build/toolchain/bin/golangci-lint run --config=/usr/local/google/home/yufanfei/go/src/open-match.dev/open-match/.golangci.yaml

internal/app/appmain.go:1: /usr/lib/google-golang/src/strconv/quote.go:405:12: ValidString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
pkg/pb/backend.pb.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Code generated by protoc-gen-go. DO NOT EDIT.
internal/ipb/synchronizer.pb.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Code generated by protoc-gen-go. DO NOT EDIT.
cmd/backend/backend.go:1: /usr/lib/google-golang/src/strconv/quote.go:405:12: ValidString not declared by package utf8 (typecheck)
// Copyright 2018 Google LLC
pkg/structs/structs.go:1: /usr/local/google/home/yufanfei/go/pkg/mod/github.com/golang/protobuf@v1.3.2/proto/text_parser.go:255:15: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
cmd/demo-first-match/main.go:1: /usr/lib/google-golang/src/encoding/json/fold.go:83:20: DecodeRune not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
cmd/frontend/frontend.go:1: /usr/lib/google-golang/src/strconv/quote.go:405:12: ValidString not declared by package utf8 (typecheck)
// Copyright 2018 Google LLC
internal/app/minimatch/doc.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
cmd/minimatch/main.go:1: /usr/lib/google-golang/src/unicode/utf8/utf8.go:28:8: expected ';', found b00000000 (typecheck)
// Copyright 2019 Google LLC
cmd/mmlogic/mmlogic.go:1: /usr/lib/google-golang/src/strings/strings.go:980:20: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2018 Google LLC
cmd/scale-backend/main.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
cmd/scale-frontend/main.go:1: /usr/lib/google-golang/src/encoding/json/fold.go:83:20: DecodeRune not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/app/swaggerui/swaggerui.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
cmd/swaggerui/swaggerui.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
cmd/synchronizer/synchronizer.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2018 Google LLC
pkg/harness/evaluator/golang/doc.go:1: /usr/lib/google-golang/src/strings/strings.go:980:20: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
pkg/harness/function/golang/doc.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/filter/testcases/testcases.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/testing/fake_frontend.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/util/testing/context.go:1: /usr/lib/google-golang/src/bytes/reader.go:98:18: DecodeRune not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
tools/certgen/certgen.go:1: /usr/lib/google-golang/src/strings/strings.go:980:20: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
tools/reaper/reaper.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/app/backend/backend.go:1: /usr/local/google/home/yufanfei/go/pkg/mod/github.com/golang/protobuf@v1.3.2/proto/text_parser.go:255:15: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/app/frontend/frontend.go:1: /usr/lib/google-golang/src/regexp/regexp.go:967:22: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/app/mmlogic/mmlogic.go:1: /usr/local/google/home/yufanfei/go/pkg/mod/github.com/stretchr/testify@v1.3.0/assert/assertions.go:173:18: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/app/synchronizer/evaluator_client.go:1: /usr/lib/google-golang/src/regexp/regexp.go:967:22: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/config/config.go:1: /usr/lib/google-golang/src/bytes/reader.go:98:18: DecodeRune not declared by package utf8 (typecheck)
// Copyright 2018 Google LLC
internal/filter/filter.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/logging/logging.go:1: /usr/lib/google-golang/src/bytes/reader.go:98:18: DecodeRune not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/rpc/clientcache.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/rpc/testing/helpers.go:1: /usr/lib/google-golang/src/strings/strings.go:980:20: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/set/set.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2018 Google LLC
internal/signal/signal.go:1: /usr/lib/google-golang/src/bytes/reader.go:98:18: DecodeRune not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/statestore/instrumented.go:1: /usr/lib/google-golang/src/path/filepath/match.go:215:15: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/statestore/testing/consts.go:1: /usr/lib/google-golang/src/strconv/quote.go:405:12: ValidString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/telemetry/configz.go:1: /usr/lib/google-golang/src/strings/strings.go:980:20: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/testing/e2e/common.go:1: /usr/lib/google-golang/src/fmt/scan.go:1200:16: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
internal/util/doc.go:1: /usr/lib/google-golang/src/path/match.go:200:15: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
test/e2e/fetch_matches_test.go:1: /usr/local/google/home/yufanfei/go/pkg/mod/github.com/stretchr/testify@v1.3.0/assert/assertions.go:173:18: DecodeRuneInString not declared by package utf8 (typecheck)
// +build !e2ecluster
tools/certgen/internal/internal.go:1: /usr/lib/google-golang/src/strconv/quote.go:405:12: ValidString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
tools/certgen/testing/certbundle.go:1: /usr/local/google/home/yufanfei/go/pkg/mod/github.com/stretchr/testify@v1.3.0/assert/assertions.go:173:18: DecodeRuneInString not declared by package utf8 (typecheck)
// Copyright 2019 Google LLC
tools/reaper/internal/internal.go:1: /usr/lib/google-golang/src/strings/strings.go:980:20: DecodeRuneInString not declared by package utf8 (typecheck)
```

**Comment by yfei1** [comment]
Closed by #823
